app/assets/stylesheets/thredded/components/_topic-header.scss

```diff
@@ -60,3 +60,67 @@
   font-size: $thredded-font-size-small;
   color: $thredded-secondary-text-color;
 }
+
+
+@mixin following {
```

Comment:
Sass mixins all live in one global namespace, so ours should be prefixed with `thredded-`.
However, looks like these mixins do not accept any arguments, so they probably should just be placeholder rules.

How about `%thredded-following-icon` and `%thredded-not-following-icon`?